doc/acl.md

```diff
@@ -1 +1,762 @@
 # ACL Module
+Module | HTTP Method | URL | Description 
+--- | --- | --- | ---
+[List](#index) | GET | /access | Get All List Access
+[Add](#add) | POST | /access | Add New Record to Acos
+[View](#view) | GET | /access/:id | View detail Acos by id
+[Edit](#edit) | PUT | /access/:id | Edit Existing record in Acos by id
+[Delete](#delete) | DELETE | /access/:id | Delete Existing record in Acos by id
+[Grant](#grant) | POST | /access/grant/:id | insert new record in aros_acos. aco_id in path url. aro_id grab from header email
+[Revoke](#revoke) | POST | /access/revoke/:id | remove existing record in aros_acos. aco_id in path url. aro_id grab from header email.
+
+## <a name="add"></a>Add Access
+
+### Endpoint 
+POST /access
+
+### Database
+![](./acl_model.png)
+
+### Headers
+Key | Value 
+--- | ---
+Content-Type | application/json
+Accept | application/json
+Email | cdc_user@gmail.com
```

Comment:
maybe it's my bad in previously, but, at POST /access, it's doesn't email header. Because there is no filtering data in the endpoint. so remove email on header. thanks.